### DISCUSSION TITLE
HelloGitHub Badge

### DISCUSSION THREAD

**Discussion by 521xueweihan** [opendilab/DI-engine#804] [General]
🎉 Congratulations! Your [project](https://hellogithub.com/repository/175c1e13739c4e429d0abf2b32ec583d) has been featured and recommended by the HelloGitHub community. We invite you to join the HelloGitHub Badge Program. Joining will grant you the following privileges:

- **Community Recognition**: The badge indicates that your project has successfully passed the HelloGitHub community's stringent selection and recommendation process.
- **Increased Exposure**: Displaying the badge will draw more traffic to your project, attracting additional users and contributors.
- **Enhanced Interaction**: Users can quickly understand your project through the badge and engage with it (like, comment, bookmark).
- **Feedback Collection**: Gather genuine feedback from a broad user base, aiding in the continuous improvement of your project.
- **Special Identification**: Once verified, your comments will feature a distinctive mark and be prioritized for pinning.

📌 [Click here](https://hellogithub.com/repository/175c1e13739c4e429d0abf2b32ec583d/embed) to wear the badge and join the HelloGitHub Badge Program, allowing your open-source project to shine even brighter.

---

**HelloGitHub** is a community focused on discovering, sharing, and promoting open-source projects. Since its inception in 2016, it has grown from a monthly newsletter into a dynamic community with over **10,000** users. Our footprint extends across multiple content platforms, earning the trust and support of **500,000** fans worldwide.

<img width="1245" alt="截屏2024-06-12 09 29 42" src="https://github.com/No-Chicken/OV-Watch/assets/8255800/7ac6b71e-f052-4f44-88b1-5c772e917348">

**Comment by TuTuHuss** [comment]
Dear 521xueweihan,
Thank you so much for the opportunity to join the HelloGitHub badge program! We're honored to be a part of it. We've already updated the badge in our [README](https://github.com/opendilab/DI-engine/pull/805) and look forward to your feedback. Looking forward to contributing more to the community!

Best regards,

Norman

**Reply by 521xueweihan** [reply]
Thanks, DI-engine has been pinned in the [HelloGitHub](https://hellogithub.com/) community.（1 day）

<img width="1053" alt="截屏2024-06-18 09 44 15" src="https://github.com/vastsa/FileCodeBox/assets/8255800/f4dc5ba5-16f0-4db0-8f44-d755a1ee16c9">
